src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -565,6 +565,32 @@ private void OnSyncWithDefinitionEnd(NodeModel nodeModel)
         {
             hasNodeInSyncWithDefinition = false;
         }
+        public void DispatchOnUIThread(Action a, bool blocking = false)
```

Comment:
Comments thanks!

Reply:
Does this need to be public now?

Reply:
Agreed, I will see about making it internal.